lib/src/components/NxNavigationSidebar/NxNavigationSidebar.scss

```
@@ -0,0 +1,160 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+@import '../../scss-shared/nx-container-helpers';
+
+$nx-sidebar-width__open: 255px;
+$nx-sidebar-width__closed: 72px;
+$nx-sidebar-logo-height: 34px;
+$nx-sidebar-link-height: 40px;
+
+$nx-sidebar-text-color: $nx-white;
+$nx-sidebar-background-color: $nx-blue-1000;
+$nx-sidebar-active-color: $nx-indigo-400;
+$nx-sidebar-hover-color: $nx-indigo-600;
+$nx-sidebar-selected-color: $nx-indigo-800;
+$nx-sidebar-separator-color: $nx-grey-900;
+
+.nx-page-sidebar--navigation {
+  @include container-vertical;
+
+  background-color: $nx-sidebar-background-color;
+  color: $nx-sidebar-text-color;
+  display: flex;
+  flex: none;
+  flex-direction: column;
+  width: $nx-sidebar-width__open;
+  padding: $nx-spacing-l 0 $nx-spacing-xxl 0;
+}
+
+.nx-sidebar__header {
+  @include container-vertical;
+
+  display: flex;
+  flex-direction: column;
+  margin: 0 $nx-spacing-l;
+}
+
+.nx-sidebar__toggle {
+  align-self: flex-end;
+  border-radius: 50%;
+  color: $nx-sidebar-text-color;
+  height: 41px;
+  margin: 0;
+  width: 41px;
+
+  &:hover, &:focus {
+    background-color: $nx-sidebar-hover-color;
+    border-color: $nx-sidebar-hover-color;
+  }
+
+  &:active {
+    background-color: $nx-sidebar-active-color;
+    border-color: $nx-sidebar-active-color;
+  }
+}
+
+.nx-sidebar__product-info {
+  @include container-horizontal;
+
+  align-items: center;
+  display: flex;
+  color: $nx-sidebar-text-color;
+  text-decoration: none;
+}
+
+.nx-sidebar__logo {
+  height: $nx-sidebar-logo-height;
+  margin-right: $nx-spacing-xs;
+}
+
+.nx-sidebar__content {
+  flex: 1 0 auto;
+  font-size: $nx-font-size-xs;
+  margin: $nx-spacing-l 0 0;
+
+  a {
+    color: $nx-sidebar-text-color;
+  }
+}
+
+.nx-sidebar__navigation {
+  border-bottom: 1px solid $nx-sidebar-separator-color;
+  flex: 0 0 auto;
```

Simpler and equivalent
```suggestion
  flex: none;
```